### ISSUE TITLE
Can't re/set admin room

### ISSUE THREAD

**Issue by RoiArthurB** [matrix-org/matrix-hookshot#971]
## Issue

Pretty much as the title says.

I stupidly left my HS's admin room and I can't join it again. So I looked at the documentation and issues here and found a pretty similar issue:  #579 

So, as [recommended by AndrewFerr](https://github.com/matrix-org/matrix-hookshot/issues/579#issuecomment-1600900373) in it, I removed the room, added back the bot in a new conversation, rebooted the server entirely; but it doesn't fix it and doesn't display any error in the logs anymore...

```
$ journalctl -fu matrix-hookshot
Sep 16 19:13:50 DietPi systemd[1]: Starting matrix-hookshot.service - A bridge between Matrix and multiple project management services, such as GitHub, GitLab and JIRA....
Sep 16 19:13:50 DietPi matrix-hookshot[66365]: Error response from daemon: No such container: matrix-hookshot
Sep 16 19:13:50 DietPi matrix-hookshot[66398]: Error response from daemon: No such container: matrix-hookshot
Sep 16 19:13:50 DietPi matrix-hookshot[66416]: 6513d71b9d74a04db855fedaed8649e26b997432698e01fde43ee53526b9d0ae
Sep 16 19:13:51 DietPi systemd[1]: Started matrix-hookshot.service - A bridge between Matrix and multiple project management services, such as GitHub, GitLab and JIRA..
Sep 16 19:13:56 DietPi matrix-hookshot[66469]: INFO 12:13:56:301 [ListenerService] Listening on http://0.0.0.0:9000 for webhooks
Sep 16 19:13:56 DietPi matrix-hookshot[66469]: INFO 12:13:56:308 [ListenerService] Listening on http://0.0.0.0:9003 for widgets
```

The only command working is the general `!hookshot help` but nothing else (see screenshot below).

![image](https://github.com/user-attachments/assets/01efd1dd-4138-442c-89de-d70756e3d38b)

---

How can I fix this issue ? Namely, found a way to either fully reset the bot, or more simply set a new admin room address to be able to administrate it.

**Comment by Bl4ckT0ny** [comment]
Hi @RoiArthurB ! Looks  like you  are using docker containers. Are there any reverse proxy? Are you using docker compose?  As for my  experience, I used docker-compose and nginx as  reverse-proxy. So,  if  I  restart hookshot container  only it stops to receive notifications from  homeserver.

TLDR:
* check your homeserver logs
* if you use reverse-proxy - reload it after the hookshot container reloaded
